app/controllers/orders/active_items.js

```diff
@@ -18,12 +27,51 @@ export default detail.extend({
     "sortProperties"
   ),
 
+  setDefaultSortingColumn: Ember.computed("dropDownItems", {
+    get(key) {
+      return ORDER_SORTING_OPTIONS[0];
+    },
+    set(key, value) {
+      return value;
+    }
+  }),
+
+  getStates() {
+    const utilities = this.get("utilityMethods");
+    const state = utilities.stringifyArray(this.getFilteredStates());
+    return {
+      state
+    };
+  },
+
+  getFilteredStates() {
+    return _.filter(this.get("states"), ["enabled", true]).map(
+      ({ state }) => state
+    );
+  },
```

Comment:
I think you can just make a `filteredStates` computed property, and you wouldn't need an observer to listen and set it